files/metricspusher.sh

```
@@ -23,10 +23,15 @@ while true; do
   if ! metrics=$(curl -s --max-time 10 -H "X-Auth-Token: ${HOPRD_API_TOKEN}" "http://hoprd:3001/api/v3/node/metrics"); then
     echo "Error: Failed to fetch metrics from Hoprd API"
-    sleep 5
-    continue
   fi
 
+  if ! hopr_safe_allowance=$(curl -s --max-time 10 -H 'accept: application/json' -H "X-Auth-Token: ${HOPRD_API_TOKEN}" "http://hoprd:3001/api/v3/account/balances" | jq -r '.safeHoprAllowance'); then
+    echo "Error: Failed to fetch Hopr node balances"
+  fi
+
+  # Append the new metric entry
+  metrics+="\nhopr_safe_allowance ${hopr_safe_allowance}"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, += is undefined.

(SC3024)